components/Events.js

```diff
@@ -62,6 +89,15 @@ export const Events = [
 		end: new Date(2022, 10, 18, 18, 0, 0),
 		description: "REALLY COOL DESCRIPTION THAT WE NEED TO FIGURE OUT",
 	},
+	{
+		title: "Running a Coding YouTube Channel: Interview with Namanh Kapur",
+		location: "WCH 205/206",
+		color: filters["Meetings"],
+		start: new Date(2022, 10, 22, 16, 0, 0),
+		end: new Date(2022, 10, 22, 18, 0, 0),
+		description:
+			"Youtube has over a million coding videos and counting, with thousands of CS professionals posting content about learning how to code or how to surpass and interview for a MANGA company. Join us for our final virtual GM as we hold an interview with Youtube content creator and Ex-Microsoft engineer Namanh Kapur, discussing his journey towards making videos on Youtube, his advice towards future software engineers entering the industry, and his cool tips on starting a Youtube channel and making content.",
```

Comment:
Join Namanh Kapur, a content creator, discuss his journey and various challenges he faced along the way.